### PR TITLE
Config cleanup

### PR DESCRIPTION
**Pull request by helenst**
**_This is a merge of work from the JiscRDSS fork (see #664) - ~it's WIP~ as it's just at the point where it deploys with Ansible - it's not tested yet and quite possibly doesn't work._**

MCPServer, MCPClient and Dashboard have been changed in this commit with the
following goals:

- Stop sharing configuration between components.
- Use django.conf to manages config.
- Manage config defaults within code.
- Look up environment variables:
  - ARCHIVEMATICA_DASHBOARD_*
  - ARCHIVEMATICA_MCPSERVER_*
  - ARCHIVEMATICA_MCPCLIENT_*

dbsettings, clientConfig.conf or serverConfig.conf are still considered if they
exist - they will be merged with the default INI in code. Environment strings
are applied last.

Client scripts are now executed using MCPClient's `settings.common`. So now
when they look up `django.conf` they're not looking at the Dashboard config.
It wasn't correct to assume that its config was available.

### CODE REVIEW

**Comment by helenst** [comment]
This now successfully runs a transfer so I'm de-WIPing it for review. https://github.com/artefactual-labs/ansible-archivematica-src/pull/115 required for Ansible deploy.

**Comment by sevein** [comment]
@helenst, great job! I'm having a meeting with @jrwdunham today to introduce him to these changes. I'm hoping we can get this merged very soon! Thank you!

File: src/dashboard/src/components/accounts/views.py [hunk 1]
```diff
@@ -63,9 +62,6 @@ def add(request):
 
 
 def edit(request, id=None):
-    if get_client_config_value('kioskMode') == 'True':
-        return redirect('main.views.forbidden')
-
```
Comment (jrwdunham) [inline src/dashboard/src/components/accounts/views.py:68]: Why was the `kioskMode` config param removed?

**Review by jrwdunham — CHANGES_REQUESTED** [review]
I reviewed the changes and ran the core AM acceptance tests on a VM deployed with these changes (and deployed via https://github.com/artefactual-labs/ansible-archivematica-src/pull/115) and they passed.

After rebasing this on current qa/1.x, I believe you will have to add a call to
``setup_dicts(mcpclient_settings)`` in the new policyCheck.py client script.

I also am wondering why the kioskMode var was removed.

After that, I think this is good to merge.

File: src/dashboard/src/components/accounts/views.py (hunk 1 above)
Comment (sevein) [reply src/dashboard/src/components/accounts/views.py:68]: Undocumented and unused. I think this was used at some point when we didn't have non-super users.

**Comment by jrwdunham** [comment]
Also, these changes cause 3 of the tests to fail in archivematicaCommon/tests/test_dicts.py. I think that should be fixed.

File: src/dashboard/src/components/accounts/views.py (hunk 1 above)
Comment (jrwdunham) [reply src/dashboard/src/components/accounts/views.py:68]: Ok. Thanks sevein.

**Comment by helenst** [comment]
@jrwdunham Thanks! I've fixed those tests so that they work now.

**Review by sevein — APPROVED** [review]

**Comment by jrwdunham** [comment]
@helenst Thanks! Now, unless I'm wrong, I think you just need to rebase and add the `setup_dicts` call in the new client script policyCheck.py.

**Comment by helenst** [comment]
@jrwdunham It's all rebased now and I've updated the policycheck script.

**Review by jrwdunham — APPROVED** [review]

**Pull request merged**